From a11ca2707b4b1330d3e4c83c6cf37295909bd0fc Mon Sep 17 00:00:00 2001
From: Ryan McCabe <rmccabe@redhat.com>
Date: Mon, 15 Jan 2018 09:23:09 -0500
Subject: [PATCH] sysconfig: Fix traceback

Fix a typo that caused a traceback in some situations that was introduced
as part of the fix for rhbz#1489270

Signed-off-by: Ryan McCabe <rmccabe@redhat.com>
Resolves: rhbz#1489270
---
 cloudinit/net/sysconfig.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cloudinit/net/sysconfig.py b/cloudinit/net/sysconfig.py
index bff844cd..3c5d0615 100644
--- a/cloudinit/net/sysconfig.py
+++ b/cloudinit/net/sysconfig.py
@@ -359,7 +359,7 @@ class Renderer(renderer.Renderer):
                         iface_cfg['DOMAIN'] = subnet['dns_search']
 
                 if 'dns_nameservers' in subnet:
-                    if isinstance(subnet['dns_search'], (list, tuple)):
+                    if isinstance(subnet['dns_nameservers'], (list, tuple)):
                         # Currently limited to 3 entries per resolv.conf(5)
                         dns_list = subnet['dns_nameservers'][:3]
                         for i, k in enumerate(dns_list, 1):
-- 
2.14.3

